src/languages/postgresql/postgresql.formatter.ts

```diff
@@ -128,18 +128,18 @@ const tabularOnelineClauses = expandPhrases([
   'CREATE FOREIGN TABLE',
   'CREATE [OR REPLACE] FUNCTION',
   'CREATE GROUP',
-  'CREATE INDEX',
-  'CREATE LANGUAGE',
+  'CREATE [UNIQUE] INDEX',
+  'CREATE [OR REPLACE] [TRUSTED] [PROCEDURAL] LANGUAGE',
   'CREATE OPERATOR',
   'CREATE OPERATOR CLASS',
   'CREATE OPERATOR FAMILY',
   'CREATE POLICY',
-  'CREATE PROCEDURE',
+  'CREATE [OR REPLACE] PROCEDURE',
   'CREATE PUBLICATION',
   'CREATE ROLE',
-  'CREATE RULE',
-  'CREATE SCHEMA',
-  'CREATE SEQUENCE',
+  'CREATE [OR REPLACE] RULE',
+  'CREATE SCHEMA [AUTHORIZATION]',
+  'CREATE {TEMPORARY | TEMP | UNLOGGED} SEQUENCE',
```

Comment:
I _think_ this is the right syntax. Please compare with https://www.postgresql.org/docs/17/sql-createsequence.html

Reply:
Not quite. The `[ { TEMPORARY | TEMP } | UNLOGGED ]` in postgres docs is really equivalent to just: `[ TEMPORARY | TEMP | UNLOGGED ]`.